test/test.js

```diff
@@ -12,7 +12,28 @@ const implsDir = path.resolve(__dirname, "implementations");
 const outputDir = path.resolve(__dirname, "output");
 
 beforeAll(() => {
-  const transformer = new Transformer();
+  const transformer = new Transformer({
+    processCEReactions(code) {
+      const preSteps = this.addImport("../CEReactions", "preSteps");
+      const postSteps = this.addImport("../CEReactions", "postSteps");
```

Comment:
Maybe consider:
```suggestion
      const csReactions = this.addImport("../CEReactions");
```

Since&nbsp;imports aren't&nbsp;de‑duplicated.

Reply:
Sounds reasonable. Fixed.